tests/contrib/psycopg/test_psycopg.py

```diff
@@ -87,6 +87,35 @@ def assert_conn_is_traced(self, tracer, db, service):
         eq_(span.meta["out.port"], TEST_PORT)
         eq_(span.span_type, "sql")
 
+    def test_opentracing_propagation(self):
+        # ensure OpenTracing plays well with our integration
+        query = "SELECT 'tracing'"
+        db, tracer = self._get_conn_and_tracer()
+        ot_tracer = init_tracer('psycopg-svc', tracer)
```

Comment:
@Kyle-Verhoog service names are usually separated by `-` instead of `_`. We don't have any restriction, but it's a nice way to see the service name.